org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
```

Comment:
Got it. I shall merge and fix it as well.

---
In reply to: [68286096](https://github.com/openT2T/translators/pull/22#discussion_r68286096) [](ancestors = 68286096)